src/applications/personalization/profile-2/components/ProfileWrapper.jsx

```diff
@@ -1,88 +1,123 @@
-import React, { useEffect } from 'react';
+import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
 
 import RequiredLoginView from 'platform/user/authorization/components/RequiredLoginView';
 import backendServices from 'platform/user/profile/constants/backendServices';
+import {
+  createIsServiceAvailableSelector,
+  isMultifactorEnabled,
+} from 'platform/user/selectors';
 
 import {
   fetchMilitaryInformation as fetchMilitaryInformationAction,
   fetchHero as fetchHeroAction,
+  fetchPersonalInformation as fetchPersonalInformationAction,
 } from 'applications/personalization/profile360/actions';
+import { fetchPaymentInformation as fetchPaymentInformationAction } from 'applications/personalization/profile360/actions/paymentInformation';
 
 import ProfileHeader from './ProfileHeader';
 import ProfileSideNav from './ProfileSideNav';
 
-const ProfileWrapper = ({
-  children,
-  fetchFullName,
-  fetchMilitaryInformation,
-  showLoader,
-  user,
-}) => {
-  useEffect(
-    () => {
-      fetchMilitaryInformation();
-      fetchFullName();
-    },
-    [fetchMilitaryInformation, fetchFullName],
-  );
+class ProfileWrapper extends Component {
```

Comment:
I converted this from a functional component to a class because shallow-mounting it doesn't work with the `useEffect` hook I was using. And doing a full mount of the component created a mess because the `RequiredLoginView` this renders is a connected component and I didn't want to mock a bunch of stuff to make that component happy.